src/main/java/com/shaneschulte/mc/clanarena/arena/SetupArena.java

```diff
@@ -0,0 +1,225 @@
+                barText.append(" spawns left (\"");
+                barText.append(MsgUtils.Colors.SUCCESS);
+                barText.append("add");
+                barText.append(MsgUtils.Colors.INFO);
+                barText.append("\")");
+            } else {
+                barText.append(MsgUtils.Colors.HIGHLIGHT);
+                barText.append("&lTeam2&r");
+                barText.append(MsgUtils.Colors.INFO);
+                barText.append(": ");
+                barText.append(MsgUtils.Colors.VARIABLE);
+                barText.append(team2SpawnsLeft);
+                barText.append(MsgUtils.Colors.INFO);
+                barText.append(" spawns left (\"");
+                barText.append(MsgUtils.Colors.SUCCESS);
+                barText.append("add");
+                barText.append(MsgUtils.Colors.INFO);
+                barText.append("\")");
+            }
+        }
+
+        return barText.toString();
+    }
+
+    /**
+     * Hides the SetupArena
+     * todo: change to complete() and save + initialize the arena
+     */
+    public void hide() {
+        this.bossBar.removeAll();
+        this.setArenaState(ArenaSetupState.COMPLETE);
+    }
+
+    // QOL checks
+    public boolean isDoneSelectingRegion() {
+        return (regionPos1 != null && regionPos2 != null);
+    }
+
+
+    // Getters & Setters
+    public void setArenaState (ArenaSetupState arenaState) {
+        this.arenaState = arenaState;
+        refreshBossBar();
+    }
+
+    public ArenaSetupState getArenaState () {
+        return arenaState;
+    }
+
+    public String getArenaName() {
+        return arenaName;
+    }
+
+    public void setRegionPos1(Location regionPos1) {
+        this.regionPos1 = regionPos1;
+        refreshBossBar();
+    }
+
+    public void setRegionPos2(Location regionPos2) {
+        this.regionPos2 = regionPos2;
+        refreshBossBar();
+    }
+
+    public void addFirstTeamSpawn(Location location) {
+        firstTeamSpawns.add(location);
+        refreshBossBar();
+
+        if (firstTeamSpawns.size() > minSpawns -1) MsgUtils.sendMessage(owner, MsgUtils.Colors.VARIABLE + "" +
+            firstTeamSpawns.size() + MsgUtils.Colors.INFO + " spawns for team1. Type \"" + MsgUtils.Colors.SUCCESS + "next" +
+            MsgUtils.Colors.INFO + "\" to move on");
+    }
+
+    public void addSecondTeamSpawn(Location location) {
+        secondTeamSpawns.add(location);
+        refreshBossBar();
+
+        if (secondTeamSpawns.size() > minSpawns -1) MsgUtils.sendMessage(owner, MsgUtils.Colors.VARIABLE + "" +
+            secondTeamSpawns.size() + MsgUtils.Colors.INFO + " spawns for team2. Type \"" + MsgUtils.Colors.SUCCESS + "done" +
+            MsgUtils.Colors.INFO + "\" when done");
+    }
+
+    public boolean canMoveOnFromSettingFirstTeamSpawns() {
+        return (firstTeamSpawns.size() >= minSpawns) && (secondTeamSpawns.size() <= 0);
+    }
+
+    public boolean canMoveOnFromSettingSecondTeamSpawns() {
```

Comment:
better name

Reply:
will just remove when moved to SetupArena logic